### PR TITLE
support python 3.9

### PR DESCRIPTION
**Pull request by lntuition**
We solved problem that pypika with python 3.9 version(https://github.com/kayak/pypika/pull/398)
And also pypika check python 3.9 with CI, so I think we can support python 3.9 too

### CODE REVIEW

**Comment by lntuition** [comment]
Also i'm not sure that it works with Travis well .. Check and help plz. Thank u :smile:

**Comment by lntuition** [comment]
hmm.. ci failed with mypy. Maybe is it related with https://github.com/python/mypy/issues/8627 ?

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - 3.7
   - 3.8
+  - 3.9-dev
```
Comment (grigi) [inline .travis.yml:5]: Consider replacing this with:
```yaml
matrix:
  include:
  - python: 3.9-dev
    env: TEST_RUNNER=green
    script: green
    after_success: echo
```
instead so we skip the linting until mypy gets fixed?

Comment (grigi) [reply .travis.yml:5]: I edited the yaml above, use `make testall` instead, and drop the ENV. Copy Pasta mistake.

Comment (lntuition) [reply .travis.yml:5]: That's good. I will try it !

**Comment by grigi** [comment]
:tada: !

**Pull request merged**